docs/schemaform/form-config.md

```diff
@@ -0,0 +1,341 @@
+          // Objects that start with view: will not be sent, but their children will be merged
+          // into the parent object and will be sent
+          // These can be used to remove fields from what is sent to the server, like if we have
+          // a question that is only used to reveal other questions
+          'view:field2': {
+            type: 'string'
+          }
+        }
+      },
+      
+      // Object containing the uiSchema for the page. Follows the format in the react-jsonschema-form
+      // docs, with some vets.gov specific additions. See below.
+      uiSchema: {
+        'ui:title': 'My form',
+        field1: {
+          'ui:title': 'My field'
+        }
+      }
+    }
+  }
+}
+```
+
+The `schema` and `uiSchema` objects should have similar structure. In other words, they should have the same fields organized the same way. The main difference between the structure of the two objects is that the uiSchema object does not have to contain all the fields that the schema object does and it does not need a `properties` object for sub-fields. So given this schema, for example:
+
+```js
+{
+  type: 'object',
+  properties: {
+    field1: {
+      type: 'string'
+    }
+  }
+}
+```
+
+The matching uiSchema would be:
+
+```js
+{
+  'ui:title': 'My form',
+  field1: {
+    'ui:title': 'My field'
+  }
+}
+```
+
+This does not apply to array fields; for those, you still need to specify an `items` object that contains the fields for each row in the array in the uiSchema:
+
+```js
+{
+  'ui:title': 'My form',
+  toursOfDuty: {
+    items: {
+      branchName: {
+        'ui:title': 'Branch'
+      }
+    }
+  }
+}
+```
+
+### uiSchema configuration
+In addition to the uiSchema options listed in the library docs, we have some additional options that are supported for all forms:
+
+```js
+{
+  // We use this instead of the title property in the JSON Schema
+  'ui:title': '', 
+  
+  // We use this instead of the description property in the JSON Schema. This can be
+  // a string or a React component and would normally be used on object fields in the
+  // schema to provide description text or html before a block of fields
+  'ui:description': '' || DescriptionComponent,
+  
+  // Customize the field or widget you're using
+  'ui:field': '' || FieldComponent,
+  'ui:widget': '' || WidgetComponent,
+  
+  // This widget will be shown on the review page. Should always be used if you specify
+  // a custom widget component, but can be used with regular widgets as well. Currently
+  // only implemented for string fields
+  'ui:reviewWidget': WidgetComponent,
+
+  // Use this to provide a function to make a field conditionally required. First
+  // argument is the current form data and the second is the formContext object,
+  // which will contain the form data for other pages (tbd). You should avoid making
+  // a field required in the JSON schema and using `ui:required` on the same field.
+  'ui:required': function (pageData) {
```

Comment:
Just going to remove mention of the second arg, because it isn't actually passed anymore.